Header header
string name
string parent_name
geometry_msgs/PoseWithCovariance pose
